main.c

```diff
@@ -2,8 +2,36 @@
 #include <stdio.h>
 #include <unistd.h>
 #include <pthread.h>
-
-
+#include <termios.h>
+
+struct termios term; 
+ 
+void disableTerminalEcho () { 
+  // define a terminal configuration data structure 
+ 
+  // copy the stdin terminal configuration into term 
+  tcgetattr( fileno(stdin), &term ); 
```

Comment:
Please be consistent with spaces inside the brackets across the whole program.

Reply:
@ikakok Done